src/components/create-task-modal/create-task-modal.component.coffee

```diff
@@ -46,7 +46,7 @@ angular.module('mnoUiElements').component('mnoCreateTaskModal', {
     # Private
 
     # Strip time & zone from local date and create a UTC date at 00:00 hours.
-    parseAsUtcDate = (date)->
+    parseAsUTCDate = (date) ->
       dateStr = moment(date).format('YYYY-MM-DD')
       moment.utc(dateStr).toISOString()
```

Comment:
About the dates formatting: For the moment let's keep this format. The `2.0` of `mno-enterprise-angular` is localized so it will need to use the `angular-moment` filters in the template:
```<span title="{{subscription.end_date | amUtc | amLocal}}">{{subscription.end_date | amCalendar}}</span>```

Reply:
`parseAsUTCDate` is duplicated, maybe should use a helper service.